spec/decorators/offence_decorator_spec.rb

```diff
@@ -185,5 +187,44 @@
       it { expect { call }.not_to raise_error }
       it { is_expected.to eql('mode of trial reason is a string') }
     end
+
+    context 'when the mode of trial reason code means the description should be hidden' do
+      before do
+        allow(offence).to receive(:mode_of_trial_reasons).and_return(mot_reason_ostruct_collection)
+      end
+
+      [1, 2, 6].each do |code|
+        context "when exactly one reason exists, with code #{code}" do
+          let(:mode_of_trial_reason_array) do
+            [{ code: code,
+               description: 'Defendant elects trial by jury' }]
```

Comment:
for the tests i'd use either a real code/description pair that are to be hidden, or a description something like 'to be hidden'. I think "Defendant elects trial by jury" is one that is to be shown

```suggestion
               description: 'Indictable-only offence' }]
```

Reply:
👍 have amended spec to use 'description to be hidden'